### DISCUSSION TITLE
Respository removed from HACS

### DISCUSSION THREAD

**Discussion by zlakes01** [marcokreeft87/room-card#347] [General]
![image](https://github.com/user-attachments/assets/6dd088a4-9bde-4061-af4f-77b968b7d8af)
What is going on?

**Comment by marcokreeft87** [comment]
You have to ask them why read only repositories are removed

**Reply by zlakes01** [reply]
OK I see where the problem is. But how it affects the users? Should I skip remove and use it as usual?

**Reply by marcokreeft87** [reply]
Yes you can still use it as normal

**Reply by jhrath** [reply]
I accidentally clicked the suggested remove button, and now all my front pages are not working.
How can I reinstall this repository?

**Reply by stefbex** [reply]
> I accidentally clicked the suggested remove button, and now all my front pages are not working. How can I reinstall this repository?

Having the same issue. I tried a manual installation as shown on https://github.com/marcokreeft87/room-card/wiki but this did not work.
